src/features/landing/components/ForWhoSection/ForWhoSection/ForWhoSection.tsx

```diff
@@ -0,0 +1,131 @@
+import { Box, Grid } from '@mui/material';
+import { useMemo, useState } from 'react';
+
+import useScreenSize from '../../../hooks/useScreenSize/useScreenSize';
+import scrollToRegistrationSection from '../../../utils/helpers/scrollToRegistrationSection';
+import ForWhoImagesContent from '../ForWhoImagesContent/ForWhoImagesContent';
+import ForWhoMainTextsContent from '../ForWhoMainTextsContent/ForWhoMainTextsContent';
+import ForWhoSectionCardItem from '../ForWhoSectionCardItem/ForWhoSectionCardItem';
+import ForWhoSectionCards from '../ForWhoSectionCards/ForWhoSectionCards';
+import ForWhoSectionCardsMobile from '../ForWhoSectionCardsMobile/ForWhoSectionCardsMobile';
+
+const images = {
+  mainImage: {
+    src: '/assets/img/ForWho/MainTable.png',
+    title: 'Main Table Image',
+  },
+  secondaryImage: {
+    src: '/assets/img/ForWho/AboutWallets.png',
+    title: 'About Wallets Image',
+  },
+};
+
+const CARD_ITEMS = [
+  {
+    id: 'card_1',
+    imageSrc: '/assets/img/ForWho/Ruby.png',
+    imageAltText: 'Ruby 1',
+    text: 'for_who.card_text_1',
+  },
+  {
+    id: 'card_2',
+    imageSrc: '/assets/img/ForWho/Ruby.png',
+    imageAltText: 'Ruby 2',
+    text: 'for_who.card_text_2',
+  },
+];
+
+const styles = {
+  mainBox: {
+    margin: '0 auto 0 auto',
+    padding: '56px 0 206px 0',
+    backgroundColor: '#FBFBFB',
+    height: '100%',
+    maxHeight: '44.875rem', // 718px
+    overflow: 'visible',
+  },
+  mainBoxLaptopOrLower: {
+    padding: '0 34px 206px 34px',
+  },
+  mainBoxMobileOrLower: {
+    paddingTop: '0',
+    marginTop: '0',
+  },
+  secondaryBoxContainer: {
+    maxWidth: '1192px',
+    margin: '0 auto',
+    height: '100%',
+    position: 'relative',
+  },
+  mainGrid: {
+    height: '39.8125rem', // 637px
+    width: '100%',
+    maxWidth: '74.5rem', // 1192px
+    padding: '0 0 0 0',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  mainGridMobileOrSmaller: {
+    padding: '32px 42px 38px 15px',
+  },
+};
+
+export default function ForWhoSection() {
+  const [cardItems] = useState(CARD_ITEMS);
+  const { isSmallest, isMobile, isLaptop, isTablet } = useScreenSize();
+  const handleTryItOutButtonClick = () => {
+    scrollToRegistrationSection();
```

Comment:
why you use this function for scrolling to the some section? tag a give you possibility to do this only from HTML, this is bad approach
https://www.w3docs.com/snippets/html/how-to-create-an-anchor-link-to-jump-to-a-specific-part-of-a-page.html